src/plugins/expressions/public/render.ts

```diff
@@ -29,8 +29,9 @@ import { getRenderersRegistry } from './services';
 export type IExpressionRendererExtraHandlers = Record<string, any>;
 
 export interface ExpressionRenderHandlerParams {
-  onRenderError: RenderErrorHandlerFnType;
-  renderMode: RenderMode;
+  onRenderError?: RenderErrorHandlerFnType;
+  renderMode?: RenderMode;
+  hasCompatibleActions?: (event: ExpressionRendererEvent) => Promise<boolean>;
```

Comment:
we should expose this on render component as well

Reply:
Yes, I've exposed it on React component, too.